exporter/otlpexporter/config_test.go

```diff
@@ -114,3 +114,26 @@ func TestUnmarshalInvalidConfig(t *testing.T) {
 	}
 
 }
+
+func TestConfigValidation(t *testing.T) {
```

Comment:
Instead of a new function we normally test the config via `TestUnmarshalConfig`/`TestUnmarshalInvalidConfig`.  You can add new test scenarios to `testdata/` and then add scenarios to the appropriate test function.

Reply:
I see, that make sense. Moved the new test scenario into the testdata